recipes-kernel/linux/5.4/linux-openxt_5.4.64.bb

```diff
@@ -49,3 +49,42 @@ SRC_URI += "${KERNELORG_MIRROR}/linux/kernel/v${PV_MAJOR}.x/linux-${PV}.tar.xz;n
 LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
 SRC_URI[kernel.md5sum] = "28a29677c102b211359439f78d1f3d6a"
 SRC_URI[kernel.sha256sum] = "b9d3c2938466f388a70fd190fd6691baa8b757393b267e9f7b06c4730d85d5ef"
+
+do_configure_append_openxt-installer(){
+    # vglass supports radeon and nouveau DRM.
+    kernel_conf_variable DRM_RADEON m
```

Comment:
We can just update all the relevant defconfigs.

Reply:
Agreed. It was somewhat convenient to isolate the changes initially.
Just for documentation: With https://github.com/OpenXT/xenclient-oe/pull/1383 it would become configuration fragments.